web/docs/language/features.md

```diff
@@ -7,7 +7,7 @@ title: Features
 There can be only one declaration of `app` type per Wasp project.
 It serves as a starting point and defines global properties of your app.
 
-```css
+```c
```

Comment:
Do we want to swap from `css` to `c` syntax here @vincanger? I think historically we used `css` for wasp syntax as it was closest to that. I would check with @Martinsos or @matijaSos for more on that. 👍🏻

Reply:
What @shayneczyzewski said -> but gave it a quick spin and I see that in this case `c` actually gives better result! It often depends on the specific code example. I am guessing Vince found all these examples to work better with `c`, so let's do it!